### DISCUSSION TITLE
Chicago Crime Data Environment Setup Issue

### DISCUSSION THREAD

**Discussion by Ivan-Keli** [DevSkillsHQ/discusssions#79] [General]
Hello,

I want to complete my assessment but i'm currently having difficulties try to set up my Google colab environment to run the Chicago Crime Data SQL database.

If you send me an email I can send you the notebook, or let me get a link to it.

Kindly Assist

**Comment by Ivan-Keli** [comment]
Here is the link to it;

https://colab.research.google.com/drive/1H-9UI5xdTGdie1NPMtV1gG0fYpia2tDv?usp=sharing

**Comment by ehsan-alva** [comment]
@Ivan-Keli Could you please describe the challenges you're encountering? This will allow us to provide more targeted assistance.

**Reply by Ivan-Keli** [reply]
Hey,
I'm unable to connect or access the database.

**Reply by ehsan-alva** [reply]
Hey Ivan!
As I checked, the database is accessible. Configuring the SQL environment is a mandatory step in the challenge.

**Reply by Ivan-Keli** [reply]
Hey Ehsan,

Thank you for the clarification, let me get on with it.
